MapboxNavigation/FeedbackCollectionViewCell.swift

```diff
@@ -36,6 +44,7 @@ class FeedbackCollectionViewCell: UICollectionViewCell {
     func commonInit() {
         setupViews()
         setupConstraints()
+        circleColor = .black
```

Comment:
Instead of black, can we have the default tint color be the tint color of the surrounding view controller?